### ISSUE TITLE
Should have a pluggable stemmer interface

### ISSUE THREAD

**Issue by sambitdash** [johnmyleswhite/TextAnalysis.jl#57]
Julia should have a pluggable stemmer interface supporting pos tagging or at least lemmtizer support like "Wordnet"

Native Julia porter2 stemmer is already available at: https://github.com/mguzmann/CorpusTools/blob/master/src/PortStemmer.jl

**Comment by aviks** [comment]
That code is presumably only English, we use Snowball C which supports many languages. It way be worthwhile to contribute a pure julia back end to Snowball, rather than writing a stemmer by hand.

**Comment by sambitdash** [comment]
Yes. That one is English only. I was interested in a multiple stemmer support as in NLTK which you can plug-in. A lemmatizer like Wordnet will be a good value add. I was not in the opinion that we move out of Snowball.

**Comment by aviks** [comment]
A Wordnet based stemmer should be relatively easy to do based on https://github.com/jbn/WordNet.jl
